odoo/tools/convert.py

```diff
@@ -98,9 +98,13 @@ def _eval_xml(self, node, env):
             if f_search:
                 idref2 = _get_idref(self, env, f_model, self.idref)
             q = safe_eval(f_search, idref2)
-            ids = env[f_model].search(q).ids
-            if f_use != 'id':
-                ids = [x[f_use] for x in env[f_model].browse(ids).read([f_use])]
+            records = env[f_model].search(q)
+            if f_use == 'Model':
+                ids = records
+            elif f_use == 'id':
+                ids = records.ids
+            else:
+                ids = records.mapped(f_use)
```

Comment:
Maybe I'm an idiot but I don't understand why "remove chart template" would add stuffs to the ORM

Reply:
Put in another commit

Reply:
I still don't understand what's going on, and that is also because I'm not an expert. Can you provide an example in the commit message? Also you say "some functions" but I would need to know which ones are using this piece of code. Also, changes to the ORM require tests

Reply:
I added tests now.
https://github.com/odoo/odoo/blob/6b305f86dc3d6953927d67979f2f1a28a523570c/odoo/addons/test_convert/tests/test_convert.py#L69-L84
It is used here for instance
https://github.com/odoo/enterprise/blob/faa4cda234e99ab5a9e5dfcf31ca6b8496a02509/l10n_be_hr_payroll_account/data/l10n_be_hr_payroll_account_data.xml#L47

Reply:
Why don't you use this already supported form for the example above?
```xml
<value model="res.company" eval="obj().search([('chart_template', '=', 'be')])"/>
```

Reply:
Because I didn't think of it